manage_breast_screening/notifications/management/commands/helpers/message_batch_helpers.py

```diff
@@ -29,6 +28,8 @@
     # Issue with backend services
     504,
 ]
+VALIDATION_ERROR_STATUS_CODE = 400
+MESSAGE_PATH_REGEX = r"(?<=\/messages\/)(\d*)(?=\/)"
```

Comment:
Sorry for the second-pass comment on this but I think this can be made more exact as we will always be capturing the index from the same path pattern of `/data/attributes/messages`
Also by using `\d+` in the capture group we always match at least one digit whereas `\d*` is zero or more.

I have tested the following regex against a variety of examples:

```python
r"^\/data\/attributes\/messages\/(\d+)\/[\w\/]+$"
```